cmd/dep/init.go

```diff
@@ -237,6 +238,15 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 	packages := make(map[string]bool)
 	notondisk := make(map[gps.ProjectRoot]bool)
 	ondisk := make(map[gps.ProjectRoot]gps.Version)
+
+	printNewDeP := func(pr gps.ProjectRoot, sm *gps.SourceMgr) {
+		message := "Cloning"
```

Comment:
Might be better as "Cloned", past tense, because this will only print after the sm is done pulling the dep down.